tests/tools/test_features_ranking.py

```diff
@@ -0,0 +1,277 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+import ehrapy as ep
+from ehrapy.tools import _utils
+
+
+class TestHelperFunctions:
```

Comment:
```suggestion
class TestRankFeaturesGroupsHelperFunctions:
```

extremely verbose like some Java shit, but precise...